int16 start_layer
uint8[] input
---
uint8[] output

